apertium_apy/apy.py

```diff
@@ -178,11 +185,17 @@ def apply_config(args, parser, apy_section):
 
 
 def parse_args(cli_args=sys.argv[1:]):
+    if(found is False):
+        location = langNamesDefault
+    else:
+        path = pkg_resources.resource_filename('apertium_apy', langNamesDefault)
+        location = os.path.join(os.path.dirname(os.path.dirname(path)), langNamesDefault)
     parser = argparse.ArgumentParser(description='Apertium APY -- API server for machine translation and language analysis')
     parser.add_argument('pairs_path', help='path to Apertium installed pairs (all modes files in this path are included)')
     parser.add_argument('-s', '--nonpairs-path', help='path to Apertium tree (only non-translator debug modes are included from this path)')
     parser.add_argument('-l', '--lang-names',
-                        help='path to localised language names sqlite database (default = langNames.db)', default='langNames.db')
+                        help='path to localised language names sqlite database (default = %s)' % (langNamesDefault),
```

Comment:
This isn't the right default anymore.

Reply:
Yeah I didn't noticed that. I'll change that.

Reply:
help has been changed to correct path